lib/extensions/inherited_nested_layouts/action_controller/base.rb

```diff
@@ -58,4 +58,15 @@ def self.class_layout(klass, self_)
     layout = layout_method.bind(self_)
     layout.call
   end
+
+  # Overrides {ActionController::Rendering#render} to keep track of the :layout rendering option.
+  def render(*args)
+    options = args.extract_options!
+    if options.try(:key?, :layout)
```

Comment:
Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.